erigon-lib/types/block.go

```diff
@@ -744,6 +748,35 @@ type RawBlock struct {
 	Body   *RawBody
 }
 
+func (r RawBlock) EncodingSize() int {
+	headerLen := r.Header.EncodingSize()
```

Comment:
Hmm. This doesn't look right to me. Assuming that a block is [encoded](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#block-encoding-and-validity) as
```
block = [header, transactions, ommers, withdrawals]
```
the logic should be something like
```
headerLen := r.Header.EncodingSize()
bodyLen := r.Body.EncodingSize()
payloadSize := headerLen + bodyLen
return payloadSize + rlp.ListPrefixLen(payloadSize)
```

Reply:
I, see, my bad.